hathor/transaction/block.py

```diff
@@ -147,7 +147,18 @@ def _get_previous_feature_activation_bit_counts(self) -> list[int]:
         return parent_block.get_feature_activation_bit_counts()
 
     def get_next_block_best_chain_hash(self) -> Optional[bytes]:
```

Comment:
In earlier implementation I have used this method hoping to traverse the blockchain from higher height to lower hight and it gave my the opposite. I found the description I little bit counter intuitive and decided to improve its description to let clear the in which direction a traverse would go.